modules/addons/spawn-area.lua

```diff
@@ -104,17 +120,22 @@ end
 local function spawn_entities(surface, position)
     position = apply_offset(position, config.entities.offset)
     for _, entity in pairs(config.entities.locations) do
-        local pos = apply_offset(position, { x=entity[2], y=entity[3] })
-        entity = surface.create_entity{name=entity[1], position=pos, force='neutral'}
-        if config.entities.protected then protect_entity(entity) end
+        local pos = apply_offset(position, {x=entity[2], y=entity[3]})
+        entity = surface.create_entity{name=entity[1], position=pos, force='spawn'}
```

Comment:
Force being "spawn" prevents the chests from being openned
```suggestion
        entity = surface.create_entity{name=entity[1], position=pos, force='neutral'}
```